tests/cli/test_suite.py

```diff
@@ -2209,17 +2209,15 @@ def test_suite_edit_interactive_batch_request_without_datasource_json_file_raise
     empty_data_context_stats_enabled,
 ):
     context = empty_data_context_stats_enabled
-    monkeypatch.chdir(os.path.dirname(context.root_directory))
+    monkeypatch.chdir(os.path.dirname(context.root_directory))  # noqa: PTH120
 
     project_dir: str = context.root_directory
-    uncommitted_dir: str = os.path.join(project_dir, "uncommitted")
+    uncommitted_dir: str = os.path.join(project_dir, "uncommitted")  # noqa: PTH118
 
     expectation_suite_name: str = "test_suite_name"
 
     # noinspection PyUnusedLocal
-    suite: ExpectationSuite = context.add_expectation_suite(
```

Comment:
Do we not want to save the suite returned from add_expectation_suite in these 2 places?